### PR TITLE
feat: Add panels for new skr-webhook metrics

### PR DESCRIPTION
**Pull request by lindnerby**
**Description**

Changes proposed in this pull request:

- Add new panels and row "SKR-Webhook" to overview dashboard for watcher_* runtime metrics

**Related issue(s)**
* https://github.com/kyma-project/runtime-watcher/issues/146

### CODE REVIEW

File: config/grafana/overview.json [hunk 1]
```diff
@@ -1646,7 +1646,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "watcher_listener_request_duration",
```
Comment (nesmabadr) [inline config/grafana/overview.json:1649]: Why was this removed?

Comment (lindnerby) [reply config/grafana/overview.json]: Good catch! Was a mistake, fixed

**Review by nesmabadr — APPROVED** [review]
